tests/sqllogictests/suites/query/02_function/02_0066_function_try_to_decimal

```diff
@@ -0,0 +1,14 @@
+statement ok
+CREATE TABLE test (a double) ENGINE=Memory
+
+statement ok
+INSERT INTO test VALUES (0.0941881516339449), (0.8833429252655464)
+
+query F 
+SELECT try_to_decimal(20, 19)(a) FROM test;
+----
+0.0941881516339448960
+0.8833429252655463424
```

Comment:
this test is cast a float to decimal128, how about adding a test for decimal256? 
for example:
```sql
SELECT try_to_decimal(72, 70)(a) FROM test;
```

Reply:
```rust
---- decimal::test_float_to_128 stdout ----
thread 'decimal::test_float_to_128' panicked at 'assertion failed: `(left == right)`

Diff < left / right > :
<100000000000000000000
>170141183460469231731687303715884105727
```
unit test  can not pass, do you have any ideas?

Reply:
It looks like the expected value is wrong, `i128::MAX` is `170_141_183_460_469_231_731_687_303_715_884_105_727i128`, which is not equal to `1e20`, can we change the test case?

Reply:
@b41sh  the code actullay use u64

Reply:
`1e20` is not equal to `i128::MAX`, we need to change the expected value.

Reply:
@zenus Any progress? Maybe we can just delete this test case.

Reply:
i will update today.